pkg/scc/scc.go

```diff
@@ -284,6 +465,26 @@ func (builder *Builder) WithUsers(users []string) *Builder {
 	return builder
 }
 
+// WithVolumes adds list of volumes to SecurityContextConstraints.
+func (builder *Builder) WithVolumes(volumes []securityV1.FSType) *Builder {
```

Comment:
Can you please endit this functiton? volumes size validation is missing. Please take a look below:

// WithVolumes adds list of volumes to SecurityContextConstraints.
func (builder *Builder) WithVolumes(volumes []securityV1.FSType) *Builder {
	if valid, _ := builder.validate(); !valid {
		return builder
	}

	glog.V(100).Infof("Redefining SecurityContextConstraints %s with"+
		"volumes: %v", builder.Definition.Name, volumes)

	if len(volumes) == 0 {
		glog.V(100).Infof("SecurityContextConstraints 'volumes' argument cannot be empty")

		builder.errorMsg = "SecurityContextConstraints 'volumes' cannot be empty list"
		
		return builder
	}
	
	if builder.Definition.Volumes == nil{
		builder.Definition.Volumes = volumes
		
		return builder
	}

	builder.Definition.Volumes = append(builder.Definition.Volumes, volumes...)
	
	return builder
}

Reply:
np, only one question: last time you ask me to replace 
`if builder.Definition.Volumes == nil{`
with
`if len(builder.Definition.Volumes) == 0 {`
is it still relevant?

Reply:
I meant **volumes** argument not **builder.Definition.Volumes**. I just copy/pasted wrong string. 
In this comment https://github.com/openshift-kni/eco-goinfra/pull/129#discussion_r1336273234 I shared that I made a mistake.

Reply:
Please revert it back to **if builder.Definition.Volumes == nil** instead of **if len(builder.Definition.Volumes) == 0 {**